### ISSUE TITLE
Twitter clone - Uninstall next-auth. Will use firebase auth later instead.

### ISSUE THREAD

**Issue by Bungycode** [Bungycode/twitter#27]
### Summary
Uninstall next-auth. Will use firebase auth later instead.
 
### Tasks
List of things that are involved with accomplishing this 

- [x] Use npm to uninstall the "next-auth" package and resolve any merge conflicts.

### Acceptance Criteria
Checklist of functionality that must be true for this to be considered done, and things that should be verified when reviewing PRs for this issue.

- [x] next-auth package is successfully uninstalled.
- [x] Merge into development branch.

### Blocks
Links to all issues that this issue blocks, and any needed explanation
* ...

### Blocked By
Links to all issues that this issue is dependent on, and any needed explanation
* ... 

> Include Any attachments or links to resources needed to complete this task.
> Link to target repo, assign feature, and labels as necessary.
